pages/index.js

```diff
@@ -2,8 +2,8 @@ import DeviceList from "@/components/DeviceList/deviceList";
 
 export default function Home({ devices }) {
   return (
-    <main>
+    <div>
       <DeviceList devices={devices} />;
-    </main>
+    </div>
```

Comment:
Do you really need a `<div>` for this case? Hint: Your `<div>`is surrounding just one element.